### ISSUE TITLE
[bug] - getTeam from example - TypeError: Cannot read property 'split' of undefined

### ISSUE THREAD

**Issue by mavrick** [gigobyte/HLTV#40]
```javascript
HLTV.getTeam({id: 6137}).then(res => {
    console.log(res);
})
```
```
(node:25940) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: Cannot read property 'split' of undefined
(node:25940) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by gigobyte** [comment]
HLTV are changing their design every couple of months, thanks for reporting those issues, it's hard to keep checking everything.

**Comment by mavrick** [comment]
travis or circle ci have free tiers for public repos, maybe we can write some units tests and some end to end tests that could run periodically to catch these kind of things?

**Comment by gigobyte** [comment]
Yeah, I know about that, it's certainly a possibility. PRs are always welcome :)

**Comment by gigobyte** [comment]
Fixed the issue in v2.6.1
